src/zm_db.cpp

```diff
@@ -37,23 +37,35 @@ void zmDbConnect()
   my_bool reconnect = 1;
   if ( mysql_options( &dbconn, MYSQL_OPT_RECONNECT, &reconnect ) )
     Fatal( "Can't set database auto reconnect option: %s", mysql_error( &dbconn ) );
-  std::string::size_type colonIndex = staticConfig.DB_HOST.find( ":/" );
-  if ( colonIndex != std::string::npos )
+  std::string::size_type colonIndex = staticConfig.DB_HOST.find( ":" );
```

Comment:
See my reply here https://github.com/ZoneMinder/ZoneMinder/pull/1498#issuecomment-238396576.

I put it in the wrong spot =/